superbuild/CMakeLists.txt

```diff
@@ -1096,7 +1097,7 @@ if (NOT BUILD_STAGE2_TOOLCHAIN_ONLY)
     if (NOT STAGE2_ISPC_INSTALL_PREFIX STREQUAL "NO")
         add_custom_target(install-stage2-ispc
             COMMENT "Install stage2 to ${STAGE2_ISPC_INSTALL_PREFIX}"
-            COMMAND ${CMAKE_COMMAND} -E copy_directory ispc-stage2 ${STAGE2_ISPC_INSTALL_PREFIX}
+            COMMAND ${CMAKE_COMMAND} -E copy_directory ${ISPC_STAGE2_INSTALL_PREFIX} ${STAGE2_ISPC_INSTALL_PREFIX}
```

Comment:
What is the difference between `${ISPC_STAGE2_INSTALL_PREFIX}` and `${STAGE2_ISPC_INSTALL_PREFIX}`? When should be used one or another?

Reply:
Yes, naming here is quite misleading. So, [this patch](https://github.com/ispc/ispc/pull/2582/commits/c6cf7e1e272158df36220a87c2373abcc954d056) adds the support of installing the output of superbuild procedure to the outer folder. 
`ISPC_STAGE2_INSTALL_PREFIX` is internal variable and should not be used by user, its value either `ispc-stage2` folder inside `build` folder or value from `CMAKE_INSTALL_PREFIX` which user should use.

`STAGE2_ISPC_INSTALL_PREFIX` was a user provided var to generate custom target to install only `stage2-ispc`. Probably, this special target is obsolete now when the superbuild takes into consideration `CMAKE_INSTALL_PREFIX`. Let me check that and adjust if needed.